### ISSUE TITLE
prometheusremotewriteexporter missing in collector?

### ISSUE THREAD

**Issue by mentos1386** [aws-observability/aws-otel-collector#1163]
**Describe the bug**
I was trying to use `prometheusremotewriteexporter` but the collector fails.

**Steps to reproduce**
Try to use `prometheusremotewriteexporter` exporter.

**What did you expect to see?**
Working `prometheusremotewriteexporter` exporter.

**What did you see instead?**
A message that it's not available, and if i meant to use the `awsprometheusremotewrite` instead.

**Additional context**
I'm trying to push to not aws hosted prometheus, so i don't need aws sigv4 authentication. How can i push metrics to a prometheus server?

**Comment by bryan-aguilar** [comment]
Sorry for the confusion. Unless you are building an image yourself the `prometheusremotewrite` exporter has not released yet and is not included in `v0.17.0`. In collector versions `v0.17.0` and earlier `awsprometheusremotewriteexporter` is the closest option. The [component list](https://aws-otel.github.io/docs/releases) on the website is a better reflection of what is included in current releases.

**Comment by dnutels** [comment]
Hi. 

The site lists `prometheusremotewrite` just the same as the docs, and yet `latest` for ADOT still fails with:

```
... application run finished with error: failed to get config: cannot unmarshal the configuration: unknown exporters type "prometheusremotewriteexporter" for "prometheusremotewriteexporter" (valid values: [dynatrace sapm datadog otlphttp awsxray awsemf awsprometheusremotewrite logzio logging otlp prometheus file signalfx])
```
